vite.config.ipfs.ts

```diff
@@ -0,0 +1,28 @@
+///<reference types="vitest" />
```

Comment:
Could we do it with an env variable instead of a whole new config?

https://vitejs.dev/guide/env-and-mode.html

Reply:
Yeah the thing is that with @cloudhead we wanted two separate build and eventually start commands, [for context see here](https://discord.com/channels/841318878125490186/856904179820920833/1039548248945332294) 
like `npm build:ipfs `vs. `npm build`

And we wanted to avoid env variables and define the routing mode in the build step. [context](https://discord.com/channels/841318878125490186/856904179820920833/1039547141359669308)

Reply:
I think we could define them as you mentioned here https://github.com/radicle-dev/radicle-interface/pull/478#discussion_r1021511935 and then keep both configs, since we also have to have a different base URL for hash routing than for history routing

Reply:
_or_ :) we could still have the env var and just do:

```diff
-    "start:ipfs": "vite -c ./vite.config.ipfs.ts",
+    "start:ipfs": "HASH_ROUTING=true vite",
     "serve": "vite preview",
     "build": "scripts/build",
-    "build:ipfs": "scripts/build-ipfs",
+    "build:ipfs": "HASH_ROUTING=true scripts/build",
```

Reply:
It would be better if configuration happens in one place/way though, not spread accross env vars and config files. Perhaps there's a way to share most of the config between the two?

Reply:
ok @rudolfs @cloudhead I think I arrived at a way of doing configuration, env vars and config files that I feel confident to die on 😅 

Feel free to challenge me ⚔️

Reply:
I like what you came up with.